pinot-common/src/main/java/org/apache/pinot/common/metrics/ValidationMetrics.java

```diff
@@ -133,124 +134,126 @@ public ValidationMetrics(PinotMetricsRegistry metricsRegistry) {
   /**
    * Updates the missing segment count gauge.
    *
-   * @param resource The resource for which the gauge is updated
+   * @param tableNameWithType The table name with type for which the gauge is updated
```

Comment:
since we are changing the comment and parameter names now: can we change to table name with type and database?

Reply:
Good question. All internal pinot references to tableName/ rawTableName / tableNameWithType are prefixed with database name by design so we are skipping specifying it in the variable name. External table name inputs are translated at entry itself to maintain the convention for internal use.